Mail/Views/Thread List/ThreadListCell.swift

```diff
@@ -101,6 +101,7 @@ struct ThreadListCell: View {
     }
 
     private func didTapCell() {
+        viewModel.addToSearchHistoryIfNeeded()
```

Comment:
I know it had already this name but maybe we can rename it to something more meaningful like `addCurrentSearchTermToHistoryIfNeeded()` it's more verbose but we understand it's not really related to the thread / cell being tapped.